### PR TITLE
Add getfixedaddress rpc #2

### PR DESCRIPTION
**Pull request by willliw**
Add a new function: getfixedaddress, which will return the default address.
Also change the getnewaddress, now will return a brand new address. 
Can solve #2 #3 issue.

### CODE REVIEW

**Comment by l1204366** [comment]
@zhouer @hihiben @lbyyy

File: src/wallet.cpp [hunk 1]
```diff
@@ -2604,8 +2605,7 @@ string CWallet::MintMoney(int64_t nValue, CWalletTx& wtxNew, type_Color color)
         if (nValue != 1)
             return "value of color 0 must be 1";
         CPubKey pubkey;
-        if (!reservekey.GetReservedKey(pubkey))
-            return NULL;
+	pubkey = vchDefaultKey;
```
Comment (hihiben) [inline src/wallet.cpp]: alignment

**Comment by lbyyy** [comment]
pass

**Comment by l1204366** [comment]
哈這個PR後來是怎麼回事啊

**Pull request closed without merging**